pkg/container-disk/container-disk_test.go

```diff
@@ -182,8 +190,15 @@ var _ = Describe("ContainerDisk", func() {
 					},
 				})
 
-				vmi := api.NewMinimalVMI("fake-vmi")
-				appendContainerDisk(vmi, "r0")
+				vmi := libvmi.New(
+					libvmi.WithContainerDisk("r0", "someimage:v1.2.3.4"),
+					libvmi.WithResourceMemory("64M"),
+					libvmi.WithLimitMemory("64M"),
+					libvmi.WithResourceCPU("1"),
+					libvmi.WithLimitCPU("1"),
+				)
+
+				By("Setting the VMI's resource requirements")
```

Comment:
You can drop the explicit resource instructions below, since now u used libvmi.

Reply:
Looks like you reverted the libvmi options. Can you please use the appropriate options instead of modifying the VMI manually?

Reply:
make new option  `WithUID`  ??